src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    { user.name }
```

Comment:
Use destructuring for the user object as well
```suggestion
    {name}
```